Generator/tests/unit/Board.test.ts

```diff
@@ -137,22 +137,14 @@ describe("solve Boards", () => {
         expect(singleNakedSingle.getStrategyScore()).toBe(StrategyEnum.NAKED_SINGLE);
         for (let i:number = 0; i < StrategyEnum.COUNT; i++) {
             if (i === StrategyEnum.NAKED_SINGLE) {
-                expect(singleNakedSingle.getStrategies()[i]).toBeTruthy;
+                expect(singleNakedSingle.getStrategies()[i]).toBeTruthy();
             }
             else {
-                expect(singleNakedSingle.getStrategies()[i]).toBeFalsy;
+                expect(singleNakedSingle.getStrategies()[i]).toBeFalsy();
             }
         }
     });
 
-    it('should solve single naked single using hidden single', () => {
```

Comment:
Is there a reason for removing this test?

Reply:
Originally I had hidden singles placing values. Since then I have thought about what strategies can be reduced into others and decided to make hidden singles remove all the notes from their cell leaving only the hidden single which then reduces to a naked single instead of placing the value directly. I think this simplifies how we explain hidden sets as well as simplifying implementation of isHiddenSet. Why do I mention all of this? Because for a strategy to be used by a solver it must remove notes (or place value if it is naked single) but since this value was already a naked single there was no notes to remove so this test failed because even though it forced solver to find the hidden single first it didn't use it.